website/data/docs-nav-data.json

```diff
@@ -679,10 +679,6 @@
         "title": "Overview",
         "path": "builders"
       },
-      {
-        "title": "Alicloud ECS",
-        "path": "builders/alicloud-ecs"
-      },
       {
```

Comment:
This file still contains the post-processor. 
There's more than one link in the sidebar https://packer-5crayggmd-hashicorp.vercel.app/docs/post-processors/alicloud-import